### ISSUE TITLE
Special characters issue

### ISSUE THREAD

**Issue by annayafi** [geeklearningio/gl-dotnet-email#34]
When writing raw text in the *.hbs file, special characters as é, à, è aren't printed right in the email. I tried all possible encodings for the *.hbs files.  I had to use character entities such as  `&eacute;`  instead. I'm not talking about variables enclosed in braces but raw text directly in the template though.

**Comment by sandorfr** [comment]
Seems it's an unspecified charset causing this issue: https://www.emailonacid.com/blog/article/email-development/the_importance_of_content-type_character_encoding_in_html_emails/ 
Is it with sendgrid or smtp?

**Comment by annayafi** [comment]
Yes exactly. It's with smtp, I tried to specify the encoding within the template, and tried to change the `SmtpEmailProvider.cs ` class and add  ` message.Headers.Add("Content-Type", "content=text/html; charset=\"UTF-8\"");` to check if the problem could be solved that way. Outlook is still misreading special characters.
Would be ideal though if I could stick with specifying the content type in the *.hbs files, but I read here https://stackoverflow.com/questions/16255487/encoding-to-utf-8-in-email that MailMessage removes the content type tags in the html body, maybe MimeMessage would do the same...

**Comment by annayafi** [comment]
The problem is clearly coming from MimeKit, I tried a little console app and I have exactly the same problem. So we might open an issue in their repo ? 
FYI, when using System.Net.Mail, it works fine: 
```
  var message = new MailMessage("whoever@gmail.com", "whoever@gmail.com", "test", "é à è");
            message.Subject = "test";
            message.BodyTransferEncoding = System.Net.Mime.TransferEncoding.Base64;
            using (var client = new SmtpClient("127.0.0.1", 25))
            {
                client.Send(message);
            }
```

Now if you change the `BodyTransferEncoding` property to 8 bit , then we run in the same exact problem as with Mime Kit...

**Comment by asiffermann** [comment]
So we should follow [this simple fix](https://github.com/jstedfast/MimeKit/issues/424#issuecomment-412350376), and allow, maybe in a second step, to configure the chosen ContentTransferEncoding.

**Comment by asiffermann** [comment]
Fixed by #35, thanks @annayafi !
